content/docs/command-reference/remote/modify.md

```diff
@@ -758,6 +758,47 @@ more information.
 
 </details>
 
+<details>
+
+### Click for OSF
+
+- `url` - remote location:
+
+  ```dvc
+  $ dvc remote modify myremote url osf://osf.io/path/to/dir
+```
+
+- `project` - [OSF](https://osf.io/) project name. You can see it in the `url` line
+  of your project page.
+
+  ```dvc
+  $ dvc remote modify myremote project <project_name>
+  
+- `user` - OSF account username (email address):
```

Comment:
Is the username always the email address in OSF @aerubanov?

Reply:
By default, yes. But the user can change it.